prboom2/src/r_defs.h

```diff
@@ -283,13 +289,20 @@ typedef enum
   ST_NEGATIVE
 } slopetype_t;
 
-typedef byte r_flags_t;
-#define RF_TOP_TILE 0x01 // Upper texture needs tiling
-#define RF_MID_TILE 0x02 // Mid texture needs tiling
-#define RF_BOT_TILE 0x04 // Lower texture needs tiling
-#define RF_IGNORE   0x08 // Renderer can skip this line
-#define RF_CLOSED   0x10 // Line blocks view
-#define RF_ISOLATED 0x20 // Isolated line
+typedef uint16_t r_flags_t;
+#define RF_TOP_TILE 0x001 // Upper texture needs tiling
```

Comment:
Please use 4 characters for the hex to imply the full size for consistency (i.e., 0x0000)